.github/workflows/pending-changelog-checker.yml

```diff
@@ -0,0 +1,22 @@
+name: "Pending Changelog Checker"
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
+    branches:
+      - master
```

Comment:
which branches we work on for core is weird. Instead of a normal release workflow, we usually just apply the celestia-specific changes to an existing release branch of tendermint, and then keep that branch up to date/add features until we want to support a new tendermint release branch. 

meaning that we will have to change this line in each branch we want to enforce keeping track of changes in to that respective branch

Reply:
Great find. Yeah, the current version we use is on this branch:
- https://github.com/celestiaorg/celestia-core/tree/v0.34.x-celestia
- and the corresponding tag: https://github.com/celestiaorg/celestia-core/releases/tag/v0.34.14-celestia

Reply:
TBH, the workflow isn't even clear to me anymore. In the past most changes were made to master. In case we wanted to change sth in this repo today, would the PR always be against the 0.34.x branch? (and soon against the 0.35 branch?). If yes, what is the purpose of master?

Reply:
I'll update the action to check the `v0.34x-celestia` branch for now. When we move to `v0.35x-celestia` we can add that as well and maybe remove v0.34.